src/legacy/server/config/schema.js

```diff
@@ -252,6 +252,7 @@ export default () => Joi.object({
   }).default(),
 
   savedObjects: Joi.object({
+    maxImportPayloadBytes: Joi.number().default(52428800),
```

Comment:
50 mB is a pretty large default, we currently limit all requests to 1mB by default. Perhaps we should start smaller around 10mB and see how this works in the field?